### ISSUE TITLE
What NFC readers are supported?

### ISSUE THREAD

**Issue by apsigma** [nfc-tools/libnfc#656]
Is `ACR122u` the only supported NFC reader? The docs don't mention anything about supported devices, would be good.

I have a `ACR1255u-j1` that works on my Windows machine, but I have not been able to get it to work on Ubuntu 20.4 with libnfc, it says it cannot find the device. Is this expected?

```
>> nfc-list
nfc-list uses libnfc 1.8.0
No NFC device found.
```

**Comment by cheriimoya** [comment]
i doubt this reader will work. If i understood it correctly, libnfc needs the reader hardware to support direct transmit. This is not the case for my ACR1252, so i have to use PC/SC.

**Comment by mdxs** [comment]
Recently experimented with a DL533N CS - LibNFC NFC Reader from Digital Logic and using `nfcpy` (Python module using LibNFC) my code worked without modifications; in my configuration (using a Raspbeery Pi) I had to allow access to the new USB device and changed the hint to find it by replacing the Vendor ID and Product ID parts in the "usb:vid:pid" hint. However, initially I just changed "usb:vid:pid" to "usb" so that I could even swap and test the (old) AC122U and the (new) DL533N reader with my software. Found no issues so far, but YMMV.
